src/create-user/controller.ts

```diff
@@ -27,6 +27,10 @@ class CreateUserController extends Controller {
     const nickname = ctx.request.body.nickname;
     const type = ctx.request.body.type;
 
+    if (type === 'user' && !principalService.isIdentityValid(identity, type)) {
+      throw new UnprocessableEntity('Identity must include mailto:');
```

Comment:
Error is no longer correct. This should probably be something like `Identity must be a valid URI`.

One more note here, we started actually adopting some json-schema in a12nserver, so it's also possible to use that here and just fully do validation with json schema, but this is fine too.